src/clang-c-frontend/clang_c_convert_literals.cpp

```diff
@@ -63,10 +63,10 @@ bool clang_c_convertert::convert_integer_literal(
       integer2string(val.getZExtValue()),
       type);
   }
-  else
+  else // "_x=100" uses this. "100" is considered as signed by default. Same as "assert(sum>100)"
   {
     the_val = constant_exprt(
-      integer2binary(val.getSExtValue(), bv_width(type)),
+      integer2binary(val.getSExtValue(), bv_width(type)), // val.getSExtValue()=100, bv_width(type)=32
```

Comment:
Improve this comment.

Reply:
Done. 
Removed this comment. This annotation is redundant as the code is self-explanatory.